doc/whats_new/v0.21.rst

```diff
@@ -318,6 +318,11 @@ Support for Python 3.4 and below has been officially dropped.
   and now it returns NaN and raises :class:`exceptions.UndefinedMetricWarning`.
   :issue:`12855` by :user:`Pawel Sendyk <psendyk>`.
 
+- |Fix| :func:`metrics.label_ranking_average_precision_score` was returning
+  incorrect values where some samples had degenerate labels and `sample_weight`
+  was used.
```

Comment:
It seems that you're saying that `sample_weight` shouldn't be used.

How about:

> Fixed a bug in [...] where ``sample_weight`` wasn't taken into account for samples with degenerate labels

Reply:
ping @dpwe  ;) could you please address this so we can merge?

Reply:
Sorry for the delay.  I made the change you suggested.